### PR TITLE
Gasless components minor refactor

### PR DESCRIPTION
**Pull request by selankon**
## Description

Jost few minor refactors in order to let the Gasless plugin ready:

- [x] Deleted the is governance enabled on DAO settings page
- [x] Extracts and reuse the component to add multisig wallet during the DAO creation, on the execution multisig step. On this way, the component and the style is the same

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have tested my code on the test network.

### CODE REVIEW

File: src/components/addCommittee/index.tsx [hunk 1]
```diff
@@ -67,45 +75,46 @@ const AddCommittee: React.FC = () => {
...
+      <WalletTable
+        controlledWallets={controlledWallets}
+        handleAdd={handleAdd}
+        handleDeleteEntry={handleDeleteEntry}
+        handleDeleteAll={handleDeleteAll}
+        rowValidator={addressValidator}
+        walletFieldName={'committee'}
+      />
       <AlertInline
         message={t('createDAO.step3.distributionWalletAlertText') as string}
```
Comment (Fabricevladimir) [inline src/components/addCommittee/index.tsx]: Not from your PR, but can you remove that casting as a string please?

File: src/components/multisigWallets/walletTable.tsx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+  return (
+    <TableContainer>
+      {!isMobile && (
+        <TableTitleContainer>
+          <Title>{t('labels.whitelistWallets.address')}</Title>
+        </TableTitleContainer>
+      )}
+      {controlledWallets.map((field, index) => (
+        <div key={index}>
```
Comment (Fabricevladimir) [inline src/components/multisigWallets/walletTable.tsx]:
```suggestion
        <div key={field.id}>
```

File: src/components/multisigWallets/walletTable.tsx [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+            {t('labels.whitelistWallets.addAddress')}
+          </Button>
+          {/*
+          To be enabled when csv functionality is there
+          <ButtonText
+            label={t('labels.whitelistWallets.uploadCSV')}
+            mode="ghost"
+            size="large"
+            onClick={() => alert('upload CSV here')}
+          /> */}
```
Comment (Fabricevladimir) [inline src/components/multisigWallets/walletTable.tsx]: Please feel free to delete this thing.

File: src/components/multisigWallets/walletTable.tsx [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+          <Title>{controlledWallets.length}</Title>
+        </TotalWalletsContainer>
+      </SummaryContainer>
+    </TableContainer>
+  );
+};
+
+const TableContainer = styled.div.attrs(() => ({
+  className: 'rounded-xl bg-neutral-0 flex flex-col',
+}))``;
```
Comment (Fabricevladimir) [inline src/components/multisigWallets/walletTable.tsx:131]: Can you add a space after each of these?

File: src/components/multisigWallets/index.tsx [hunk 5]
```diff
@@ -199,31 +138,31 @@ const Container = styled.div.attrs(() => ({
...
 }))``;
 
-const SummaryContainer = styled.div.attrs(() => ({
+styled.div.attrs(() => ({
   className: 'flex xl:p-6 p-4 flex-col space-y-3',
 }))``;
-const TotalWalletsContainer = styled.div.attrs(() => ({
+styled.div.attrs(() => ({
   className: 'flex place-content-between',
 }))``;
```
Comment (Fabricevladimir) [inline src/components/multisigWallets/index.tsx]: These should be deleted, right?

**Review by Fabricevladimir — COMMENTED** [review]
Very nice refactor! Looking good!

File: src/components/multisigWallets/walletTable.tsx (hunk 4 above)
Comment (selankon) [reply src/components/multisigWallets/walletTable.tsx:131]: 388f70aa

File: src/components/multisigWallets/index.tsx (hunk 5 above)
Comment (selankon) [reply src/components/multisigWallets/index.tsx:168]: 0cc812a9

Comment (selankon) [reply src/components/multisigWallets/index.tsx]: 388f70aa

File: src/components/multisigWallets/walletTable.tsx (hunk 3 above)
Comment (selankon) [reply src/components/multisigWallets/walletTable.tsx]: 388f70aa

File: src/components/addCommittee/index.tsx (hunk 1 above)
Comment (selankon) [reply src/components/addCommittee/index.tsx]: 388f70aa

**Review by Fabricevladimir — APPROVED** [review]

**Pull request merged**